index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+    <script  src="js/index.js"></script>
+
+
+    <h1 id="title">Contact Form</h1>
+    <p id="description">This is our contact form, we'll be on touch!</p>
+    
+    <form id="survey-form">
+      <ul class="errorMessages"></ul>
+      
+      <label id="name-label">Name</label>
+      <input id="name" type="text" required="true" placeholder="FULL Name"><br>
+      <label id="email-label">Email</label>
+      <input id="email" type="email" required="true" placeholder="example@gmail.com"><br>
+      <label id="number-label" for="number">Number</label><br>
+      <input type="number" id="number" name="number" value="0" min="1" max="10" require="true" placeholder="0"><br>
+      <label for="cars">Choose a car:</label>
+      <select id="dropdown" name="dropdown">
+        <option value="volvo">Volvo</option>
+        <option value="saab">Saab</option>
+        <option value="fiat">Fiat</option>
+        <option value="audi">Audi</option>
+      </select><br>
+      <label><input type="checkbox" id="cbox1" value="2002"> 2002</label><br>
+
+    <input type="checkbox" id="cbox2" value="2003"> <label for="cbox2">2003</label><br>
+        <label for="textarea">Leave message</label><br>
+        <label>Employer statut</label>
+        <input type="radio" name="gender" value="employed" checked><br>
+        <input type="radio" name="gender" value="freelancer" ><br>
+        <input type="radio" name="gender" value="applier" ><br>
+        <input type="radio" name="gender" value="unemployed" >
+    
+      <textarea id="textarea" name="w3review" rows="4" cols="50">
+      
+      </textarea>
+      <div class="buttons">
+        <button id="submit">Submit</button>
+    </div>
+  </form>
+    
+    
+   
+  </body>
+</html>
```

Comment:
Good work with the code, the form it's looking very good :smiley: . But there are some errors still on linters review :sweat:, please check them and make the changes it suggests until everything it's checked in green :ok_hand: